infra/terraform/test-org/org/locals.tf

```diff
@@ -663,5 +663,12 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name        = "notebooks-blueprint-security"
```

Comment:
Any reason this is not prefixed with "terraform"?

Reply:
@g-awmalik This was an older blueprint and so it didn't follow standard naming - open to renaming the repo itself if that's possible

Reply:
Sorry for the delaying response. Let's let this one slide.